src/test/java/com/shippo/model/OrderTest.java

```diff
@@ -0,0 +1,108 @@
+package com.shippo.model;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.time.LocalDateTime;
+
+import org.junit.Test;
+
+import com.shippo.exception.APIConnectionException;
+import com.shippo.exception.APIException;
+import com.shippo.exception.AuthenticationException;
+import com.shippo.exception.InvalidRequestException;
+import com.shippo.exception.ShippoException;
+
+public class OrderTest extends ShippoTest {
+
+	@Test
+	public void testValidCreate() {
+		Order testObject = (Order) getDefaultObject();
+		assertEquals("PAID", testObject.getOrderStatus().toString());
+	}
+
+	@Test(expected = InvalidRequestException.class)
+	public void testInvalidCreate() throws AuthenticationException,
+			InvalidRequestException, APIConnectionException, APIException {
+		Order.create(getInvalidObjectMap());
+	}
+
+	@Test
+	public void testRetrieve() throws AuthenticationException,
+			InvalidRequestException, APIConnectionException, APIException {
+		Order testObject = (Order) getDefaultObject();
+		Order retrievedObject = Order.retrieve((String) testObject.getObjectId());
+		assertEquals(testObject.getObjectId(), retrievedObject.getObjectId());
+	}
+
+	@Test(expected = InvalidRequestException.class)
+	public void testInvalidRetrieve() throws AuthenticationException,
+			InvalidRequestException, APIConnectionException, APIException {
+		Order.retrieve("invalid_id");
+	}
+
+	@Test
+	public void testListAll() throws AuthenticationException,
+			InvalidRequestException, APIConnectionException, APIException {
+		OrderCollection objectCollection = Order.all(null);
+		assertNotNull(objectCollection.getData());
+	}
+
+	@Test
+	public void testListPageSize() throws AuthenticationException,
+			InvalidRequestException, APIConnectionException, APIException {
+		Map<String, Object> objectMap = new HashMap<String, Object>();
+		objectMap.put("results", "1"); // one result per page
+		objectMap.put("page", "1"); // the first page of results
+		OrderCollection OrderCollection = Order
+				.all(objectMap);
+		assertEquals(OrderCollection.getData().size(), 1);
+	}
+
+	public static Object getDefaultObject() {
+        Address addressFrom = (Address) AddressTest.getDefaultObject();
+        Address addressTo = (Address) AddressTest.getSecondObject();
+
+		Map<String, Object> lineItem = new HashMap<String, Object>();
+        lineItem.put("title", "Demo Line Item Object");
```

Comment:
more spacing inconsistencies here

Reply:
Spaces & Tabs used in different files, I've installed a linter to fix this